index.html

```diff
@@ -85,81 +85,25 @@
     <div class="flex-spacer"></div>
     <nav>
       <a class="btn rounded cta secondary play-cta" href="http://play.witheve.com">Play</a>
-      
+
       <a href="http://incidentalcomplexity.com">Blog</a>
       <a href="http://docs.witheve.com">Docs</a>
       <a href="https://groups.google.com/forum/#!forum/eve-talk">Community</a>
     </nav>
   </header>
 </div>
 
-
-	
-	
-
-
 <main class="main layer-wrapper">
-	<div class="toplayer">
-		<div class="drawer">
-			<div aria-label="Navigation">
-  
-  <div class="scrollable">
-    <div class="wrapper">
-      
-
-      
-      <div id="sidebar" class="nav-collapse">
-          <ul class="sidebar-menu">
-            
-            
-              
-                <li class="sub-menu active">
-                  <a href="http://docs.witheve.com/" class="">
-                      
-                      <span class="colored">Introduction</span>
-                      <span class="menu-arrow arrow_carrot-right"></span>
-                  </a>
-                  <ul class="sub">
-                    
-                      <li><a href="http://docs.witheve.com/install/"> Installation </a> </li>
-                      
-                    
-                      <li><a href="http://docs.witheve.com/tutorials/quickstart.eve/"> Quickstart </a> </li>
-                      
-                    
-                      <li><a href="http://docs.witheve.com/syntaxreference/"> Syntax Ref </a> </li>
-                      
-                    
-                  </ul>
-                </li>
-              
-            
-              
-                <li>
-                  <a class="" href="http://docs.witheve.com/handbook/libraries">
-                    
-                    <span class="colored">Library</span>
-                  </a>
-                </li>
-              
-            
-          </ul>
-      </div>
-
-    </div>
-  </div>
-</div>
-		</div>
-		<article class="article">
+	<div class="toplayer landing">
 			<div class="wrapper">
-
-				
-          <a href="http://docs.witheve.com/v0.2/index.html"><h2>v0.2 Documentation</h2></a>
-          <a href="http://docs.witheve.com/v0.3/index.html"><h2>v0.3 Documentation</h2></a>
-
-				
-
+        <h2>
+          Pardon the mess! We're updating our docs and have two options:
```

Comment:
This sounds too much like "early 90s construction man gif" I think. We'll keep the old docs around for posterity always, so it's really just this landing page that's temporary and needs love.